src/pages/works/{PrismicArticle.uid}.js

```diff
@@ -0,0 +1,59 @@
+import * as React from "react";
+import { graphql } from "gatsby";
+import Layout from "../../components/Layout";
+import Article from "../../components/Article";
+
+const ArticlePage = ({ data }) => {
+  if (!data) return null;
+  const {
+    prismicArticle: {
+      data: {
+        title: { text: title },
+        article_content: { richText: articleContent },
+        author: { text: author },
+        image: { url: imageUrl, alt: imageAlt },
+        image_caption: { text: imageCaption },
+      },
+    },
+  } = data;
+
+  return (
+    <Layout>
+      <Article
+        title={title}
+        content={articleContent}
+        author={author}
+        imageUrl={imageUrl}
+        imageAlt={imageAlt}
+        imageCaption={imageCaption}
+      />
+    </Layout>
+  );
+};
+
+export const articleQuery = graphql`
+  query ArticlePageQuery($uid: String) {
+    prismicArticle(uid: { eq: $uid }) {
+      data {
+        article_content {
+          richText
+        }
+        author {
+          text
+        }
+        image {
```

Comment:
it might be worth talking to designers about whether this should be used as a thumbnail by default or if we should add a separate thumbnail field in prismic; i'm just thinking about articles potentially being pieces in a project alongside videos and they'll need a thumbnail for the home/explore pages
i'm not sure if the team even plans on including articles in video series but we should anticipate it just in case

Reply:
good point I asked Holly and added a separate thumbnail field in prismic!